### PR TITLE
Updated docs with build v1.0.0-preview-008920001-990

### PR DESCRIPTION
**Pull request by vgsankar**

### CODE REVIEW

File: releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Microsoft Extensions Azure App Configuration
+
+The Azure App Configuration Extension in the Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview).
```
Comment (jimmyca15) [inline releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md]: This doesn't belong here. This is the description for the vsts task.

File: releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+### v1.0.0-preview-008920001-990
+
+* Added `TrimKeyPrefix` method to remove prefixes from the list of returned key-values from the config store.
+* Improved durability on Watch, so that transient network errors do not cause the process to detach.
+* Retries on network error when listing keys in Managed Identity Connector.
```
Comment (jimmyca15) [inline releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md]: >Retries on network error when listing keys in Managed Identity Connector [](start = 2, length = 72)

Should read `Added additional retries on network error when using `ConnectWithManagedIdentity`.

File: releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+* Enabled Correlation-Context for telemetry/request-tracing.
+    * ***Telemetry can be disabled by setting the environment variable `AZURE_APP_CONFIGURATION_TRACING_DISABLED` to `True` or `1`***
+    * Added telemetry for the following:
+        * Hosting platform.
+        * Differentiating Initialization and Watch/Observe requests.
+* Adding feature flags has been improved, and prevented removal of default query with `null` label.
```
Comment (jimmyca15) [inline releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md]: >Adding feature flags has been improved [](start = 2, length = 38)

Not sure what was improved. The point can be explained more or removed.

Comment (jimmyca15) [inline releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md]: >prevented removal of default query with `null` label [](start = 46, length = 52)

Is this related to the feature flags improvement? If so we should just say that while using feature flags, the default query is still used.

Comment (vgsankar) [reply releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md]: Yes, that is the improvement I was referring to. I've added more details.

**Comment by zhenlan** [comment]
- Can we add the date of release?
- Can we use request tracing instead of telemetry to be consistent throughout the doc?
